Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -327,6 +333,15 @@ public void Serialize<T>(IIonWriter writer, T item)
 
             if (item is object)
             {
+                var customSerializerAttribute = item.GetType().GetCustomAttribute<IonSerializerAttribute>();
+
+                if (customSerializerAttribute != null) {
+                    var customSerializerFactory = (IIonSerializerFactory)Activator.CreateInstance(customSerializerAttribute.Factory);

```

Comment:
We need to look for either a `Serializer` or a a `Factory`

Reply:
Added.